docgen/src/guides/migration.md

```diff
@@ -31,7 +31,7 @@ search.addWidget(instantsearch.widgets.searchbox({
 }));
 ```
 
-You can read more about these options [here](/widgets/searchBox.html).
+You can read more about these options [here](instantsearch.js/v2/widgets/searchBox.html).
```

Comment:
The migration page is already on `instantsearch.js/v2` therefore removing the `/` should be enough, no?

Reply:
Sorry I hadn't seen your review earlier -- thank you @bobylito @Haroenv !